frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/__mocks__/k8s-data.ts

```diff
@@ -0,0 +1,15 @@
+import { K8sModel } from '../../../api/common-types';
+
+export const DeploymentModel: K8sModel = {
```

Comment:
Minor nit, I just find it odd you are duping this model for a test. But I suppose test data is test data...

Reply:
yeah, the actual model is in the internal package so mocked it for test data.